static/js/click_handlers.js

```diff
@@ -242,6 +243,13 @@ export function initialize() {
         window.location.href = $(this).attr("href");
     });
 
+    $("body").on("click", "#scroll-to-bottom-button-container", (e) => {
```

Comment:
The hover effect and tooltip are attached to the circular icon, but this click handler is attached to the surrounding square, and can be triggered by clicking outside of the circle.

Reply:
Good catch! Thanks!